CMakeLists.txt

```diff
@@ -194,7 +194,15 @@ if (WIN32)
 else()
     set(QUIC_COMMON_FLAGS "")
     set(QUIC_COMMON_DEFINES "")
-    set(QUIC_WARNING_FLAGS -Werror -Wall -Wextra -Wformat=2 -Wno-type-limits -Wno-unknown-pragmas -Wno-unused-value -Wno-multichar CACHE INTERNAL "")
+    set(QUIC_WARNING_FLAGS -Werror -Wall -Wextra -Wformat=2 -Wno-type-limits
+        -Wno-unknown-pragmas -Wno-unused-value -Wno-multichar -Wno-switch
```

Comment:
I think I would prefer to not have `Wno-switch` be global. This should only be affecting the perf lib anyway. I found this with the mac build, and I'll look there for what our solution was.

Reply:
Nevermind, turns out for Mac we also just had to globally add this. So I'm fine with it